src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int LINE_FIRST = 0;
+    private static final int LINE_SECOND = 1;
+    private static final int LINE_THIRD = 2;
+    private static final int LINE_FOURTH = 3;
```

Comment:
```suggestion
    private static final int START_DATE_INDEX = 0;
    private static final int NAME_INDEX = 1;
    private static final int HOURS_OF_WORK_INDEX = 2;
    private static final int SALARY_PER_HOUR_INDEX = 3;
```

still not informative constant naming